recipes/nexusformat/meta.yaml

```
@@ -0,0 +1,50 @@
+{% set name = "nexusformat" %}
+{% set version = "0.4.20" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "58023fe0913a6d8273afc971cdc64ec0fc909f5d7d39a7819df0674ca2108ec2"
+
+build:
+  number: 0
+  entry_points:
+    - nxstack=nexusformat.scripts.nxstack:main
+  noarch: "python"
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - pip
+    - python
+    - setuptools
+    - numpy
+  run:
+    - h5py
+    - numpy
+    - python
+
+test:
+  imports:
+    - nexusformat
+    - nexusformat.nexus
+    - nexusformat.scripts
+  commands:
+    - nxstack --help
+
+about:
+  home: "https://nexpy.github.io/nexpy/"
+  license: "BSD 3-Clause"
```

The SPDX identifier has a dash here:
```suggestion
  license: "BSD-3-Clause"
```
(Just to be consistent with other recipes)